lib/uploadcare/errors/type_error.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Uploadcare
+  module Errors
+    class TypeError < TypeError; end
```

Comment:
Why is this class not included in the gem?

Reply:
because it is used in Rails-gem, not in ruby one